school_library.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getters
+  attr_reader :name
+
+  attr_reader :age
+
+  def add_name(value)
+    @name = value
+  end
+
+  def add_age(age)
+    @age = age
+  end
```

Comment:
- Since you are already using the getter and setter for the name and age variables, I suggest removing this part of your code. Please fix this 🙏

Reply:
Noted :+1: